Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -125,7 +125,6 @@ public Swerve(Vision vision)
 
     resetAnglesToAbsolute( );
     resetGyro(180.0); // All starting positions facing driver in field relative
-    resetOdometry(new Pose2d(0, 0, m_heading));
```

Comment:
Not sure why this is deleted in the Comp2023-imported project. Seems like it was OK when called in the constructor.